docs/_data/walkthrough_data.yml

```diff
@@ -0,0 +1,308 @@
+  paired: yes
+  readname: Standard
+  indexing: Dual
+  barcode: BC tag
+  files:
+      - standard.dual_index.paired.sam
+- format: BAM
+  quality: Standard
+  paired: yes
+  readname: Standard
+  indexing: Dual
+  barcode: BC tag
+  files:
+      - standard.dual_index.paired.bam
+- format: CRAM
+  quality: Standard
+  paired: yes
+  readname: Standard
+  indexing: Dual
+  barcode: BC tag
+  files:
+      - standard.dual_index.paired.cram
+
+# FASTQ DATA
+## normal output for ReadTools: Illumina-legacy, dual/single indexed, single-end or pair-end (split files)
+- format: FASTQ
+  quality: Standard
+  paired: no
+  readname: Illumina-legacy
+  indexing: Single
+  barcode: In Read Name
+  files:
+      - legacy.single_index.SE.fq
+- format: FASTQ
+  quality: Standard
+  paired: yes
+  readname: Illumina-legacy
+  indexing: Single
+  barcode: In Read Name
+  files:
+      - legacy.single_index.paired_1.fq
+      - legacy.single_index.paired_2.fq
+- format: FASTQ
+  quality: Standard
+  paired: no
+  readname: Illumina-legacy
+  indexing: Dual
+  barcode: In Read Name
+  files:
+      - legacy.dual_index.SE.fq
+- format: FASTQ
+  quality: Standard
+  paired: yes
+  readname: Illumina-legacy
+  indexing: Dual
+  barcode: In Read Name
+  files:
+      - legacy.dual_index.paired_1.fq
+      - legacy.dual_index.paired_2.fq
+## pair-end interleaved (less common)
+- format: FASTQ
+  quality: Standard
+  paired: yes
+  readname: Illumina-legacy
+  indexing: Single
+  barcode: In Read Name
+  files:
+      - legacy.single_index.interleaved.fq
+- format: FASTQ
+  quality: Standard
+  paired: yes
+  readname: Illumina-legacy
+  indexing: Dual
+  barcode: In Read Name
+  files:
+      - legacy.dual_index.interleaved.fq
+## casava read names
+- format: FASTQ
+  quality: Standard
+  paired: no
+  readname: Casava
+  indexing: Single
+  barcode: In Read Name
+  files:
+      - casava.single_index.SE.fq
+- format: FASTQ
+  quality: Standard
+  paired: yes
+  readname: Casava
+  indexing: Single
+  barcode: In Read Name
+  files:
+      - casava.single_index.paired_1.fq
+      - casava.single_index.paired_2.fq
+
+# ILLUMINA QUALITY: these reads are encoded with Illumina quality, which is misencoded for SAM/BAM/CRAM
+## SAM (misencoded)
+- format: SAM
+  quality: Illumina
+  paired: no
+  readname: Standard
+  indexing: Single
+  barcode: BC tag
+  files:
+      - misencoded.single_index.SE.sam
+## FASTQ
+- format: FASTQ
+  quality: Illumina
+  paired: no
+  readname: Illumina-legacy
+  indexing: Sinlge
+  barcode: In Read Name
+  files:
+      - legacy.single_index.illumina_quality.SE.fq
+- format: FASTQ
+  quality: Illumina
+  paired: yes
+  readname: Illumina-legacy
+  indexing: Single
+  barcode: In Read Name
+  files:
+      - legacy.single_index.illumina_quality_1.fq
+      - legacy.single_index.illumina_quality_2.fq
+
+# BARCODES FORMAT: these reads have dual-indexes split in two tags or in the read name
+## barcode in BC/B2 (e.g., illumina2bam output)
```

Comment:
To reduce complexity of the dataset. Only standardized and mapped is in SAM, BAM and CRAM. The rest are SAM files, because it is convenient for git to track changes and can be converted with samtools to BAM/CRAM if users want to test on that kind of files.

Should I point this out in the help page, @AnnaMariaL?